woocommerce/woocommerce-admin
# Fix #4086 misaligned 'required' text on selects

Fixes #4086

This moves the error text in the business details OBW step from within the control to below it, to bring them in  line with other controls.
 
### Screenshots

![image](https://user-images.githubusercontent.com/224531/81359789-76b6ec00-911d-11ea-8038-1011b35c56e2.png)

### Detailed test instructions:

- run this SQL against your database:

```
DELETE FROM `wp_options` WHERE option_name = 'woocommerce_onboarding_profile'
```

- start the OBW and step through to the business details step
- click to open the options for one of the fields
- click away without making a selection
- the "Required" help text should appear

### Changelog Note:

Fix: #4086 misaligned 'required' text on selects
